### PR TITLE
enhance redirects

### PR DESCRIPTION
**Pull request by nevikashah**
@GregBrimble @deadlypants1973 @Ekwuno

### CODE REVIEW

File: products/pages/src/content/platform/redirects.md [hunk 1]
```diff
@@ -41,19 +43,31 @@ Cloudflare currently offers limited support for advanced redirects. More support
...
-| Placeholders                    | No      | /blog/:year/:month/:date/:slug /news/:year/:month/:date/:slug            |
-| Query Parameters                | No      | /shop id=:id /blog/:id 301                                               |
-| Force (shadowing)               | No      | /workers/ /workers/index.html 200!                                       |
-| Domain-level redirects          | No      | workers.example.com/* workers.example.com/blog/:splat 301 |
-| Redirect by country or language | No      | / /us 302 Country=us                                                     |
+| Feature                                   | Support | Example                                                         | Notes                                  |
+| ----------------------------------------- | ------- | --------------------------------------------------------------- | ---------------------------------------|
+| Redirects (301, 302, 303, 307, 308)       | Yes     | /home / 301                                                     | 302 is used as the default status code |
```
Comment (GregBrimble) [inline products/pages/src/content/platform/redirects.md]: Might be nice to have the examples in wrapped as code e.g. `/home / 301`?

**Review by GregBrimble — APPROVED** [review]

File: products/pages/src/content/platform/redirects.md [hunk 2]
```diff
@@ -41,19 +43,31 @@ Cloudflare currently offers limited support for advanced redirects. More support
...
+| Rewrites (other status codes)             | No      | /blog/* /blog/404.html 404                                      |                                        |
+| Splats                                    | Yes     | /blog/* /blog/:splat                                            | See [Splats](#splats)                  |
+| Placeholders                              | Yes     | /blog/:year/:month/:date/:slug /news/:year/:month/:date/:slug   | See [Placeholders](#placeholders)      |
+| Query Parameters                          | No      | /shop id=:id /blog/:id 301                                      |                                        |
+| Force                                     | Yes     | /workers/ /workers/index.html                                   | Redirects are always followed, regardless of whether or not an asset matches the incoming request | 
+|Proxying                                   | No      |/blog/* https://blog.my.domain/:splat 200                        |                                        |
```
Comment (WalshyDev) [inline products/pages/src/content/platform/redirects.md]:
```suggestion
| Proxying                                  | No      | /blog/*https://blog.my.domain/:splat 200                        |                                        |
```

File: products/pages/src/content/platform/redirects.md [hunk 3]
```diff
@@ -41,19 +43,31 @@ Cloudflare currently offers limited support for advanced redirects. More support
...
 
 </TableWrap>
 
+## Matching
+Redirects execute before headers, so in the case of a request matching rules in both files, the redirect will win out.
+### Splats
+On matching, a splat (asterisk, `*`) will greedily match all characters. You may only include a single splat in the URL.
+
+The matched value can be used in the redirect location with `:splat`.
+### Placeholders
```
Comment (WalshyDev) [inline products/pages/src/content/platform/redirects.md]: IIRC you need spacing around headers or the docs engine has issues
```suggestion
## Matching

Redirects execute before headers, so in the case of a request matching rules in both files, the redirect will win out.

### Splats

On matching, a splat (asterisk, `*`) will greedily match all characters. You may only include a single splat in the URL.

The matched value can be used in the redirect location with `:splat`.

### Placeholders

```

File: products/pages/src/content/platform/redirects.md [hunk 4]
```diff
@@ -41,19 +43,31 @@ Cloudflare currently offers limited support for advanced redirects. More support
...
+| Feature                                   | Support | Example                                                         | Notes                                  |
+| ----------------------------------------- | ------- | --------------------------------------------------------------- | ---------------------------------------|
+| Redirects (301, 302, 303, 307, 308)       | Yes     | /home / 301                                                     | 302 is used as the default status code |
+| Rewrites (other status codes)             | No      | /blog/* /blog/404.html 404                                      |                                        |
+| Splats                                    | Yes     | /blog/* /blog/:splat                                            | See [Splats](#splats)                  |
+| Placeholders                              | Yes     | /blog/:year/:month/:date/:slug /news/:year/:month/:date/:slug   | See [Placeholders](#placeholders)      |
+| Query Parameters                          | No      | /shop id=:id /blog/:id 301                                      |                                        |
```
Comment (WalshyDev) [inline products/pages/src/content/platform/redirects.md]:
```suggestion
| Query Parameters                          | No      | /shop?id=:id /blog/:id 301                                      |                                        |
```

File: products/pages/src/content/platform/redirects.md [hunk 5]
```diff
@@ -41,19 +43,31 @@ Cloudflare currently offers limited support for advanced redirects. More support
...
+| Force                                     | Yes     | /workers/ /workers/index.html                                   | Redirects are always followed, regardless of whether or not an asset matches the incoming request | 
+|Proxying                                   | No      |/blog/* https://blog.my.domain/:splat 200                        |                                        |
+| Domain-level redirects                    | Yes     | workers.example.com/* workers.example.com/blog/:splat 301       | Must use HTTPS protocol                |
+| Redirect by country or language           | No      | / /us 302 Country=us                                            |                                        |
+| Redirect by cookie                        | No      | /* /preview/:splat 302 Cookie=preview                           |                                        |
 
 </TableWrap>
 
+## Matching
```
Comment (GregBrimble) [inline products/pages/src/content/platform/redirects.md]: This and all subsequent headings should be one level lower.

```
### Matching

#### Splats

#### Placeholders

### Additional Information
```

File: products/pages/src/content/platform/redirects.md (hunk 3 above)
Comment (lukeed) [reply products/pages/src/content/platform/redirects.md]: Hilarious that you know this

**Review by lukeed — APPROVED** [review]

**Comment by GregBrimble** [comment]
That most recent commit isn't correct, actually. Sorry, I missed Walshy's last comment. But it should just be a space.

File: products/pages/src/content/platform/redirects.md [hunk 6]
```diff
@@ -41,19 +43,36 @@ Cloudflare currently offers limited support for advanced redirects. More support
...
+| Redirects (301, 302, 303, 307, 308) | Yes     | `/home / 301`                                                   | 302 is used as the default status code                                                            |
+| Rewrites (other status codes)       | No      | `/blog/* /blog/404.html 404`                                    |                                                                                                   |
+| Splats                              | Yes     | `/blog/* /blog/:splat`                                          | See [Splats](#splats)                                                                             |
+| Placeholders                        | Yes     | `/blog/:year/:month/:date/:slug /news/:year/:month/:date/:slug` | See [Placeholders](#placeholders)                                                                 |
+| Query Parameters                    | No      | `/shop?id=:id /blog/:id 301`                                    |                                                                                                   |
```
Comment (GregBrimble) [inline products/pages/src/content/platform/redirects.md]:
```suggestion
| Query Parameters                    | No      | `/shop id=:id /blog/:id 301`                                    |                                                                                                   |
```

**Comment by lukeed** [comment]
Thanks for the follow-up :) Reverted via your suggestion

**Pull request merged**